HW2/P5/P5.txt

```diff
@@ -0,0 +1,118 @@
+Author: Xingchi Dai
+Harvard University
+CS205
+HW2
+Problem 5: Simulation and domain decomposition
+
+Part1: Performance results
+
+Performance:
+
+1 thread:
+5.30881548837 simulation frames per second
+5.40018050797 simulation frames per second
+5.41002962789 simulation frames per second
+5.25063781738 simulation frames per second
+5.39368750257 simulation frames per second
+5.46699252481 simulation frames per second
+
+4 threads:
+
+10.6414713292 simulation frames per second
+11.1193694709 simulation frames per second
+11.8052638541 simulation frames per second
+10.9182311353 simulation frames per second
+10.7273122537 simulation frames per second
+11.4661753208 simulation frames per second
+11.8020749211 simulation frames per second
+
+Note: parallel computing shows a good improvement on performance, almost two times better.
```

Comment:
I also got that four threads was about 2x faster. However, two threads was only a little slower than four, so I'm guessing it's some sort of scheduling overhead.